zproject/computed_settings.py

```diff
@@ -327,24 +327,17 @@ def get_dirs(self) -> List[str]:
 
 SESSION_ENGINE = "django.contrib.sessions.backends.cached_db"
 
-# Compress large values being stored in memcached; this is important
-# for at least the realm_users cache.
-PYLIBMC_MIN_COMPRESS_LEN = 100 * 1024
-PYLIBMC_COMPRESS_LEVEL = 1
```

Comment:
My guess is we'll discover through the chat.zulip.org deploy that we need this feature.

Reply:
d8e0c959bcb1250e94eeb085a49375357d38a023 suggests that chat.zulip.org should be 500ing with this PR deployed.  But it isn't.

Reply:
OK, the answer is available in https://github.com/jaysonsantos/python-binary-memcached/blob/master/bmemcached/protocol.py -- compression is built-in.  So we're good.